.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,81 @@
+name: Build Wheels
+
+# Only run on new tags starting with `v`
+on: [push]
```

Comment:
This is set to `push` for testing in the PR; I usually set to:
```yaml
on:
  push:
    tags:
      - 'v*'
```

so that it builds wheels only when I push new tags that start with `v`